src/pages/home/rightsection.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import "./section.css";
+
+const RightSection = () => {
+    return( 
+    <div id="right-sec-text">
+        <h4>World without trees </h4>
+        <ul>
+            <li>No Trees... No Humans.</li>
```

Comment:
Not a big deal for this assignment or such a small project but generally a good idea to put these list points into it’s own file or array. Separate UI logic from Data.